### PR TITLE
updated typography according to the new spec

### PR DESCRIPTION
**Pull request by Jinsoo94**
This branch updates typography based on the new spec! @emma-floored @hwigmore

### CODE REVIEW

**Comment by Jinsoo94** [comment]
https://app.clubhouse.io/floored/story/12532/update-typography

**Comment by emma-floored** [comment]
Awesome! Would you mind fixing one thing for me? The `blx-subtitle` on a dark background isn't quite what we want, I think. I made the background part of the style, but I think actually we just want like a `blx-subtitle light` or something that has the light text style that we can put over any background...

**Comment by Jinsoo94** [comment]
@emma-floored updated!

**Comment by emma-floored** [comment]
thanks!! 👍

File: blocks-styles/typography.styl [hunk 1]
```diff
@@ -94,6 +124,10 @@ a, .blx-link {
   blxLink()
 }
 
+h1.blx-XL {
+  blxHXL()
+}
+
```
Comment (hwigmore) [inline blocks-styles/typography.styl]: this is really silly and picky, but i feel like it's more clear to read and find things when the generic stuff (regular h1) comes first, so
```
h1 {
  blxH1()
}
h1.blx-XL {
  blxHXL()
}
```
(or could nest it with an `&.blx-XL`

Comment (hwigmore) [reply blocks-styles/typography.styl]: ooh i see why you did it this way. so they are ordered by size. fine to leave as is

Comment (Jinsoo94) [reply blocks-styles/typography.styl]: is it good to go?

**Pull request merged**